src/makeRobotAccountant.js

```diff
@@ -6,7 +6,24 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 1;
+  const getSum = (a) => {
+    const sum = (b) => {
+      // console.log('*count - ', count)
+      if (count <= 3 || count % 2 !== 0) {
+        count++;
+
+        return a + b;
+      }
+      count++;
+
+      return 'Bzzz... Error!';
+    };
+
+    return sum;
```

Comment:
```suggestion
    const sum = (b) => {
      count++;

      if (count <= 3 || count % 2 !== 0) {
        return a + b;
      }

      return 'Bzzz... Error!';
    };

    return sum;
```
we can increase count each time before `if` and not duplicate it

don't forget remove comments from code